centos_ci.sh

```diff
@@ -224,6 +278,16 @@ else
     cd gopath/src/github.com/minishift/minishift
     make gen_adoc_tar IMAGE_UID=$(id -u)
     docs_tar_upload $PASS
+  elif [[ "$JOB_NAME" = "minishift-release" ]]; then
+    GIT_USER="minishift-bot"
```

Comment:
maybe all in this if branch could go into a function _release()_ or _doRelease()_